src/index.js

```diff
@@ -5,7 +5,7 @@ export * from './components/Button';
 export * from './components/ButtonGroup';
 export * from './components/Card';
 export * from './components/DataTable';
-export * from './components/Form';
+  export * from './components/Form';
```

Comment:
i broke it on purpose to check if pretest correctly returns the error-code for travis